packages/stream_chat_flutter/lib/src/message_input/stream_message_input.dart

```diff
@@ -899,64 +986,85 @@ class StreamMessageInputState extends State<StreamMessageInput>
+                maxHeight: widget.maxHeight,
+                child: PlatformWidgetBuilder(
+                  web: (context, child) => Focus(
+                    skipTraversal: true,
+                    onKeyEvent: _handleKeyPressed,
+                    child: child!,
+                  ),
+                  desktop: (context, child) => Focus(
+                    skipTraversal: true,
+                    onKeyEvent: _handleKeyPressed,
+                    child: child!,
+                  ),
+                  mobile: (context, child) => Focus(
+                    skipTraversal: true,
+                    onKeyEvent: _handleKeyPressed,
+                    child: child!,
+                  ),
+                  child: Row(
+                    children: [
+                      Flexible(
+                        child: StreamMessageTextField(
+                          key: const Key('messageInputText'),
+                          maxLines: widget.maxLines,
+                          minLines: widget.minLines,
+                          textInputAction: widget.textInputAction,
+                          onSubmitted: (_) => sendMessage(),
+                          keyboardType: widget.keyboardType,
+                          controller: _effectiveController,
+                          focusNode: _effectiveFocusNode,
+                          style: _messageInputTheme.inputTextStyle?.copyWith(
+                            color: UnikonColorTheme.messageInputHintColor,
+                          ),
+                          autofocus: widget.autofocus,
+                          textAlignVertical: TextAlignVertical.center,
+                          decoration: _getInputDecoration(context),
+                          textCapitalization: widget.textCapitalization,
+                          autocorrect: widget.autoCorrect,
+                          contentInsertionConfiguration:
+                              widget.contentInsertionConfiguration,
+                        ),
                       ),
-                    ),
+                      if (_effectiveController.message.quotedMessage == null)
+                        IconButton(
+                          onPressed: () {
+                            Navigator.push(
+                                context,
+                                MaterialPageRoute(
+                                  builder: (context) =>
+                                      const GalleryPickerScreen(),
+                                ));
+                          },
+                          icon: const Icon(
+                            Icons.attachment,
+                            color: UnikonColorTheme.darkGreyColor,
+                          ),
+                        ),
+                      if (_effectiveController.message.quotedMessage == null &&
+                          _effectiveController.text.isEmpty)
+                        IconButton(
+                          onPressed: () {},
```

Comment:
Knowing this onPressed is blank or it got left by mistake. Please Look into this.

Reply:
Code pushed with UA-621 here